tensorflow/python/estimator/canned/dnn.py

```diff
@@ -486,7 +519,7 @@ def __init__(
       config=None,
       warm_start_from=None,
       loss_reduction=losses.Reduction.SUM,
-      batch_norm=False,
+      batch_norm=None,
```

Comment:
This should also support boolean for backward compatibility. 
You can do something like following in init-fn:
if batch_norm and batchPnorm is bool:
  batchnorm = experimental_batch...

Reply:
Hi @lukmanr 
I couldn't see the backward compatibility. Could you please add a test to check backward compatibility: one with batch_norm=False, one with batch_norm=True.